app/models/ingest_measurement.rb

```diff
@@ -11,5 +11,129 @@ class << self
     def data_for(query)
       search(query: query).results.map(&:_source)
     end
+
+    def query_sensor_data(device_urn) # rubocop:disable Metrics/MethodLength
+      search "query":
+                 { "bool": {
+                   "filter": [
+                     {
+                       "range": {
+                         "@timestamp": {
+                           "gte": 'now-8w',
+                           "lte": 'now'
+                         }
+                       }
+                     },
+                     {
+                       "term": {
+                         "device_urn": device_urn
+                       }
+                     }
+                   ]
+                 } },
+             "aggs": {
+               "sensor_data": {
+                 "date_histogram": {
+                   "interval": '12h',
```

Comment:
It seems that we should use `fixed_interval` instead.

```suggestion
                   "fixed_interval": '12h',
```

I got this message from elasticsearch.

```
[interval] on [date_histogram] is deprecated, use [fixed_interval] or [calendar_interval] in the future.
```